projects/bp-gallery/cypress/e2e/bulk-edit.cy.ts

```diff
@@ -0,0 +1,22 @@
+import { login, logout } from '../utils/login-utils';
+
+describe('bulk edit', () => {
+  before(() => {
+    cy.visit('http://localhost:3000/latest/');
+    login();
+  });
+  after(() => {
+    cy.wait(1000);
+    logout();
+  });
+
+  it('select multiple photos manually', () => {
+    cy.wait(500);
+    cy.get('.picture-preview').eq(0).find('.adornment.bottom-left').click();
+    cy.get('.picture-preview').eq(1).find('.adornment.bottom-left').click();
+    cy.get('.picture-preview').eq(2).find('.adornment.bottom-left').click();
+    cy.get('.picture-preview').eq(0).find('[data-testid="CheckBoxIcon"]').should('exist');
+    cy.get('.picture-preview').eq(1).find('[data-testid="CheckBoxIcon"]').should('exist');
+    cy.get('.picture-preview').eq(2).find('[data-testid="CheckBoxIcon"]').should('exist');
+  });
+});
```

Comment:
eventuell noch nen test um zu gucken ob man wenn man auf "alle bilder editieren" (oder so) drückt auch wirklich die ausgewählten Bilder im BulkEditView geöffnet werden?

Reply:
I think the bulk edit tests are way too few in general, but I think they could be expanded in another PR, so this one can finally be merged. I will open an issue.

Reply:
See #222